plugins/meta/podman-machine/main.go

```diff
@@ -36,14 +36,12 @@ func cmdAdd(args *skel.CmdArgs) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to parse config")
 	}
+
 	hostAddr, err := getPrimaryIP()
 	if err != nil {
 		return err
 	}
-	// No portmappings, do nothing
-	if len(portMaps.RuntimeConfig.PortMaps) < 1 {
```

Comment:
Why is this gone?

Reply:
The early return breaks containers that dont have portmaps because this is a chained plugin and we have to run the print of the configs to hand it off to the next one.